st2common/st2common/validators/api/reactor.py

```diff
@@ -15,7 +15,11 @@
 
 import six
 
+from apscheduler.triggers.cron import CronTrigger
```

Comment:
I mean our implementation is already quite coupled to apscheduler so it's not breaking any abstraction (imo). Or you mean something else?

And the same validation is already performed in rules engine where apscheduler jobs are created. This change just moves it up the stack so the validation is also performed earlier when rule is created.

This is something which is almost always desirable since it allows us to fail early and provide a better user-experience.